contracts/src/instances/ezeth-linea/EzETHLineaHyperdrive.sol

```diff
@@ -0,0 +1,104 @@
+///    %%%%%%%%&&             ##   ++0000+    +0000++   ##           &&%%%%%%%%%
+///         %&&&                ##  +o000+    +000o+  ##              &&&%
+///                              ## ++00+-    -+00++ ##
+///                               #% ++0+      +0++ %#
+///                               ###-:Oo.++++.oO:-###
+///                                ##: 00++++++00 :##
+///                    #S###########* 0++00+++00++0 *##########S#
+///                  #S               % $ 0+++0 $ %              S#
+///                #S   ----------   %+++++:#:+++++%-----------    S#
+///              #S   ------------- %++++: ### :++++%------------    S#
+///             S    ---------------%++++*\ | /*++++%-------------     S
+///           #S     --------------- %++++ ~W~ ++++%666--o UUUU o-      S#
+///         #S?      ---------------  %+++++~+++++%&&&8 o  \  /  o       ?S#
+///        ?*????**+++;::,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,::;+++**????*?
+///      #?+////////////////////////////////////////////////////////////////+?#
+///    #;;;;;//////////////////////////////////////////////////////////////;;;;;#
+///  S;;;;;;;;;//////////////////////////////////////////////////////////;;;;;;;;;S
+/// /;;;;;;;;;;;///////////////////////////////////////////////////////;;;;;;;;;;;;\
+/// |||OOOOOOOO||OOOOOOOO=========== __  ___        ===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|  \[__ |   \  /===========OOOOOOOO||OOOOOOOO|||
+/// |||OOOOOOOO||OOOOOOOO===========|__/[___|___ \/ ===========OOOOOOOO||OOOOOOOO|||
+/// ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+/// |||////////000000000000\\\\\\\\|:::::::::::::::|////////00000000000\\\\\\\\\\|||
+/// SSS\\\\\\\\000000000000////////|:::::0x666:::::|\\\\\\\\00000000000//////////SSS
+/// SSS|||||||||||||||||||||||||||||:::::::::::::::||||||||||||||||||||||||||||||SSS
+/// SSSSSSSS|_______________|______________||_______________|______________|SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+/// SSSSSSSS                                                                SSSSSSSS
+///
+/// @author DELV
+/// @title EzETHLineaHyperdrive
+/// @notice A Hyperdrive instance that uses a EzETHLinea vault as the yield source.
+/// @dev This instance supports the Renzo protocol on Linea. The vault shares token
+///      is the rebasing LRT token xezETH. There are a few special things about
+///      this integration:
+///
+///      - The base token address is the ETH constant.
+///      - The vault shares token address is the xezETH address.
+///      - The vault share price is provided by an oracle.
+///      - Interest accrues sporadically when the oracle is updated.
+///      - Base deposits aren't supported since there is a deposit fee for
+///        minting xezETH.
+///      - Base withdrawals aren't supported since there isn't an instantaneous
+///        way to withdraw from xezETH.
```

Comment:
Love this description

Reply:
Might also be a good place to add that `minShareReserves` and `minTransactionAmount` need to be 1e15